src/commons/eventsourcing/store.rs

```diff
@@ -643,14 +644,14 @@ where
         trace!("Trying to load aggregate id: {}", handle);
 
         let info_key = Self::key_for_info(handle);
-        let info: StoredValueInfo = self
+        let limit = self
             .kv
-            .get(&info_key)
+            .get::<StoredValueInfo>(&info_key)
             .map_err(|_| AggregateStoreError::InfoCorrupt(handle.clone()))?
-            .ok_or_else(|| AggregateStoreError::InfoMissing(handle.clone()))?;
```

Comment:
It will not treat it as an error if it is missing. The info is used to know what the last event is. Without this we will get whatever we can find on disk instead.